### PR TITLE
Making GitHub templates a bit more useful

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by jonathantorres** [comment]
Thank you for your contribution @piotr-zuralski! 
@raphaelstolt I'd like to know what you think about these templates. Thanks in advance

**Comment by raphaelstolt** [comment]
Will have a closer look once #phpnw17 is done.

File: src/stubs/github/PULL_REQUEST_TEMPLATE.stub [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+| Tests pass?   | yes/no
+| Fixes         | #... <!-- #-prefixed issue number(s), if any -->
+| License       | MIT
 
 Changes proposed in this pull request:
+
+<!--
+- Please fill in this template according to the PR you're about to submit.
+- Replace this comment by a description of what your PR is solving.
+-->
```
Comment (raphaelstolt) [inline src/stubs/github/PULL_REQUEST_TEMPLATE.stub]: Missing newline. Have you enabled EditorConfig support in your PHPStorm IDE?

File: tests/stubs/with-github-templates/pull_request_template.stub [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+| Tests pass?   | yes/no
+| Fixes         | #... <!-- #-prefixed issue number(s), if any -->
+| License       | MIT
 
 Changes proposed in this pull request:
+
+<!--
+- Please fill in this template according to the PR you're about to submit.
+- Replace this comment by a description of what your PR is solving.
+-->
```
Comment (raphaelstolt) [inline tests/stubs/with-github-templates/pull_request_template.stub]: Missing newline. Have you enabled EditorConfig support in your PHPStorm IDE?

File: .gitignore [hunk 3]
```diff
@@ -5,6 +5,9 @@
 *.sublime-project
 *.sublime-workspace
 
+# Intellij / PHPStorm
```
Comment (raphaelstolt) [inline .gitignore]: How does this relate to the GitHub templates?

File: src/stubs/github/PULL_REQUEST_TEMPLATE.stub [hunk 4]
```diff
@@ -1,3 +1,17 @@
...
+| Q             | A
+| ------------- | ---
+| Branch?       | x.y.z or master <!-- see comment below -->
+| Bug fix?      | yes/no
+| New feature?  | yes/no <!-- don't forget updating CHANGELOG.md files -->
+| BC breaks?    | yes/no
+| Deprecations? | yes/no <!-- don't forget updating UPGRADE-*.md files -->
+| Tests pass?   | yes/no
+| Fixes         | #... <!-- #-prefixed issue number(s), if any -->
+| License       | MIT
```
Comment (raphaelstolt) [inline src/stubs/github/PULL_REQUEST_TEMPLATE.stub]: Should be derived from the `--license` option value and not be statically MIT.

File: src/stubs/github/ISSUE_TEMPLATE.stub [hunk 5]
```diff
@@ -1,3 +1,17 @@
+| Q                | A
+| ---------------- | -----
+| Bug report?      | yes/no
+| Feature request? | yes/no
+| BC Break report? | yes/no
+| RFC?             | yes/no
+| version          | x.y.z
```
Comment (raphaelstolt) [inline src/stubs/github/ISSUE_TEMPLATE.stub]: s/version/Version?

File: src/stubs/github/ISSUE_TEMPLATE.stub [hunk 6]
```diff
@@ -1,3 +1,17 @@
+| Q                | A
+| ---------------- | -----
+| Bug report?      | yes/no
+| Feature request? | yes/no
+| BC Break report? | yes/no
```
Comment (raphaelstolt) [inline src/stubs/github/ISSUE_TEMPLATE.stub]: s/Break/break

File: tests/stubs/with-github-templates/pull_request_template.stub [hunk 7]
```diff
@@ -1,3 +1,17 @@
...
+| Q             | A
+| ------------- | ---
+| Branch?       | x.y.z or master <!-- see comment below -->
+| Bug fix?      | yes/no
+| New feature?  | yes/no <!-- don't forget updating CHANGELOG.md files -->
+| BC breaks?    | yes/no
+| Deprecations? | yes/no <!-- don't forget updating UPGRADE-*.md files -->
+| Tests pass?   | yes/no
+| Fixes         | #... <!-- #-prefixed issue number(s), if any -->
+| License       | MIT
```
Comment (raphaelstolt) [inline tests/stubs/with-github-templates/pull_request_template.stub:10]: Should be derived from the `--license` option value and not be statically MIT.

**Comment by raphaelstolt** [comment]
Did a quick review and might be good to merge afterwards.

**Review by raphaelstolt — CHANGES_REQUESTED** [review]

File: tests/stubs/with-github-templates/issue_template.stub [hunk 8]
```diff
@@ -1,3 +1,17 @@
+| Q                | A
+| ---------------- | -----
+| Bug report?      | yes/no
+| Feature request? | yes/no
+| BC Break report? | yes/no
+| RFC?             | yes/no
+| version          | x.y.z
```
Comment (raphaelstolt) [inline tests/stubs/with-github-templates/issue_template.stub]: s/version/Version?

**Comment by raphaelstolt** [comment]
For my liking it feels too verbose and symfony-ish, which is not a bad thing per se. I would probally delete the generated content and replace it with content in a more non-technical voice/tone.

**Comment by raphaelstolt** [comment]
Hi there @piotr-zuralski, can you squash the commits into a single one?

@jonathantorres Lgtm.

**Comment by ghost** [comment]
Hi @raphaelstolt, sure, done.

**Comment by jonathantorres** [comment]
Thanks a lot guys!

**Pull request merged**